packages/api/lib/utils.js

```diff
@@ -42,6 +42,17 @@ function deconstructCollectionId(collectionId) {
   };
 }
 
+/**
+ * Returns the collectionId that a granule expects in the format
+ * "name___version"
+ *
+ * @param {Collection} collection - A Dynamo Collection object
+ * @returns {string} The formatted collectionId
+ */
+function constructCollectionId(collection) {
+  return [collection.name, collection.version].join('___');
+}
+
```

Comment:
I remember talking about this in a previous PR but I guess I never did it.

Reply:
We don't need to define this one because it already exists in `packages/message/Collections.ts`, so we can use it from the `message` package

Reply:
Ah I definitely missed it there. I was thinking it would/should be in utils because we have `deconstructCollectionId` there. It seems a little weird to use one package for that and another for `constructCollectionId`. But, then again, the one in the `message` package is used elsewhere and I don't particularly want to replace all of those instances.

Reply:
begrudgingly updated 😄 

https://github.com/nasa/cumulus/pull/2057/commits/1eb64a78d26431e9f611b2e82d6e8cb787bf6e8e